### PR TITLE
hide participants if not logged

### PR DESCRIPTION
**Pull request by Raulnet**
hide participants if not logged

### CODE REVIEW

File: features/search/events.feature [hunk 1]
```diff
@@ -10,3 +10,15 @@ Feature:
     When I am logged as "benjyd@aol.com"
     And I am on "/evenements"
     Then the "search-city" field should contain "Marseille 3e, France"
+
+  Scenario: The display count participant need logging
+    Given the following fixtures are loaded:
+      | LoadAdherentData |
```
Comment (JeremieSamson) [inline features/search/events.feature]: You should have the following to load the fixture for all scenarios 

```
Background:
    Given the following fixtures are loaded:
      | LoadAdherentData      |
```

Comment (Raulnet) [reply features/search/events.feature]: Fixed i used the background features

File: features/search/events.feature [hunk 2]
```diff
@@ -1,12 +1,25 @@
 Feature:
+  Background:
+    Given the following fixtures are loaded:
+      | LoadAdherentData        |
+      | LoadEventData         |
```
Comment (ottaviano) [inline features/search/events.feature]: CS: align pipe's chars pls

File: templates/events/_header.html.twig [hunk 3]
```diff
@@ -51,8 +51,10 @@
             {% endif %}
         {% endif %}
         <div class="text--body text--white icon--with-text l__row--center b__nudge--top-10 committee-event-attendees event-attendees">
```
Comment (ottaviano) [inline templates/events/_header.html.twig]: why do you need to keep empty DIV block on the page?

File: templates/search/_events.html.twig [hunk 4]
```diff
@@ -66,7 +66,9 @@
                     </div>
                 {% endif %}
                 <div class="text--body text--gray icon--with-text l__row--center">
```
Comment (ottaviano) [inline templates/search/_events.html.twig]: same remark

File: features/search/events.feature (hunk 2 above)
Comment (Raulnet) [reply features/search/events.feature]: fixed

File: templates/search/_events.html.twig (hunk 4 above)
Comment (Raulnet) [reply templates/search/_events.html.twig]: removed

File: features/search/events.feature [hunk 5]
```diff
@@ -1,12 +1,25 @@
...
+  Scenario: The search city is base on user's city
 
     When I am on "/evenements"
     Then the "search-city" field should contain "Paris"
 
     When I am logged as "benjyd@aol.com"
     And I am on "/evenements"
     Then the "search-city" field should contain "Marseille 3e, France"
+
+  Scenario: The display count participant need logging
```
Comment (Remg) [inline features/search/events.feature]: As a non logged-in user, I can not see the participants count of an event

File: features/search/events.feature [hunk 6]
```diff
@@ -1,12 +1,25 @@
 Feature:
-
-  Scenario: The search city is base on user's city
+  Background:
     Given the following fixtures are loaded:
       | LoadAdherentData |
+      | LoadEventData    |
+
+  Scenario: The search city is base on user's city
```
Comment (Remg) [inline features/search/events.feature]: The search city is base**d** on user's city

File: features/search/events.feature (hunk 5 above)
Comment (Raulnet) [reply features/search/events.feature]: fixed

File: features/search/events.feature (hunk 6 above)
Comment (Raulnet) [reply features/search/events.feature]: fixed

**Pull request merged**